new smart contracts/contracts/CollectionFactory.sol

```diff
@@ -25,7 +25,7 @@ contract CollectionFactory is Ownable {
     /// @notice **Mapping** that maps a **Collection Type** to **a list of Collections of the Collection Type** that **exists on this Blockchain**
```

Comment:
change folder name here `new smart contracts` spaces are to be avoided

Reply:
done